src/main/resources/db/migration/V1_3__create_tables.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE VURDERING
+(
+    id                 SERIAL PRIMARY KEY,
+    uuid               CHAR(36)    NOT NULL UNIQUE,
+    personident        VARCHAR(11) NOT NULL,
+    created_at         timestamptz NOT NULL,
+    updated_at         timestamptz NOT NULL,
+    created_by         VARCHAR(7)  NOT NULL,
+    status             VARCHAR(20) NOT NULL,
```

Comment:
Typ `FORHANDSVARSEL` / `OPPFYLT` / `IKKE_OPPFYLT`. Kunne også kalt det `utfall`, men synes ikke det passer så bra med forhåndsvarsel. Tanker?

Reply:
`status` funker. `type` kunne man også brukt, `enum VurderingType` liksom 🤷🏼‍♂️

Reply:
Kan godt gå for `type`. `status` høres litt mer ut som noe som kan oppdateres og det er vel ikke tanken her.